.github/workflows/bsi.yml

```diff
@@ -61,6 +61,21 @@ jobs:
           path: ${{ matrix.artifacts }}
         if: ${{ matrix.artifacts != '' }}
 
+  tarball:
+    name: "Source Tarball"
```

Comment:
Do we really need a job for this? Downloading the complete files in the repository should be already available by the normal GitHub UI.

Reply:
Indeed, I didn't think of the standard button on GitHub. Though, now that it's there: My long-term goal is to have a one-click download for the entire BSI deliverable. And then it'd be great to have the tarball right there.